tinyorm/src/main/java/me/geso/tinyorm/TinyORM.java

```diff
@@ -572,4 +580,70 @@ public void setQueryTimeout(int queryTimeout) {
 	public void clearQueryTimeout() {
 		this.queryTimeout = null;
 	}
+
+	/**
+	 * Set auto-commit mode (means do with transaction or not) to connection.
+	 *
+	 * @param isAutoCommit auto-commit mode
+	 * @throws SQLException
+	 */
+	 public void setAutoCommit(boolean isAutoCommit) throws SQLException {
```

Comment:
This method makes confusion for users. I suggest to remove this method from p-r.